### DISCUSSION TITLE
Rport fork become OpenRPort

### DISCUSSION THREAD

**Discussion by alhdo** [openrport/openrport#2] [Announcements]
# 🚀 Exciting News: Introducing OpenRPort - The Future of rPort!

👋 Hey there, awesome developers!

We're thrilled to share some fantastic news with you. 🎉 We've just forked the well-loved rPort project and are taking it to new heights with OpenRPort! 🌟

🔗 **Repository:** [OpenRPort on GitHub](https://github.com/openrport/openrport)

📢 **What's Happening:**

🔄 **Forked and Thriving:** Our old rPort project isn't going anywhere; it's still alive and well. But, to bring more life to the party, we've created OpenRPort, a dynamic open-source fork of the original project.

🚧 **Brand New Front-End:** As part of this exciting journey, we're busy developing a fresh and open-source front-end that's designed to make your experience even smoother and more enjoyable.

🚀 **Expanding Services:** The reinvigorated rPort project will not only be maintained, but it will also see the resurrection of all its essential services and enhancements. So, you're in for a treat!

👉 **Get Involved:**

We're stoked about this open-source adventure, and we want you to be part of it! Join us on our GitHub repository to contribute, suggest ideas, or simply explore the magic of OpenRPort.

🙌 **Let's Collaborate:** Together, we'll shape the future of rPort. OpenRPort is here to thrive, and we can't wait to see what we can achieve with your help.

🌐 **Explore OpenRPort:** [Visit OpenRPort on GitHub](https://github.com/openrport/openrport)

📚 **Documentation:** Detailed documentation and guidelines are on their way, so stay tuned!

🎉 Thank you for your continued support and enthusiasm. Here's to the next chapter in the rPort saga!

🌟 Let's make magic happen together!

With open-source love,
The OpenRPort Team 🚀

**Comment by kezon** [comment]
Team, you asked for a positive feedback so I want to give one. I am working for two non-profits (disabled people) and I managed dozens of end stations (Win + Ubuntu) and I used rport for several years. Then this turnover happened and I was so sad as the non-profits could not afford paying to vnc... and then I found this fork and I have to give all the best credits to the team - at least for the transsition from rport to openrport. The server upgrade was a piece of cake and the upgrade of end points was also flawless.

I THANK YOU for this life saver, it is perhaps much simpler than Intune, but unlike this commercial mastodont, rport just works and does what it is expected to do.

Keep up in a good work!
